packages/postgres-queries-builder/src/queries/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './select';
+// export * from './insert';
```

Comment:
Commented code

Reply:
Normal

Reply:
Why ?

Reply:
I can remove if you want but it's going to be uncommented